js-envs-test-kit/src/main/scala/org/scalajs/jsenv/test/RunTests.scala

```diff
@@ -77,7 +77,7 @@ private[test] class RunTests(config: JSEnvSuiteConfig, withCom: Boolean) {
 
   @Test // Failed in Phantom - #2053
   def utf8Test: Unit = {
-    withRun("""console.log("\u1234")""") {
+    withRun("console.log('\u1234')") {
```

Comment:
Where does this change come from?

Reply:
It caused a fatal warning (error) in 2.13.2.